rpm/xcache.spec

```
@@ -1,6 +1,6 @@
 Name:      xcache
 Summary:   XCache scripts and configurations
-Version:   1.1.0
+Version:   1.1.1
```

We're still dealing with pre-releases for now:

```suggestion
Version:   1.1.0
```